user_docs/en/userGuide.t2t

```diff
@@ -142,6 +326,25 @@ Pressing Continue will create the portable copy.
 Once creation is complete, a message will appear telling you it was successful.
 Press OK to dismiss this dialog.
 
+
+++ Portable and Temporary Copy Restrictions ++[PortableAndTemporaryCopyRestrictions]
+
+If you want to take NVDA with you on a USB thumb drive or other writable media, then you should choose to create a portable copy.
+The installed copy is also able to create a portable copy of itself at any time. 
+The portable copy also has the ability to install itself on any computer at a later time.
+However, if you wish to copy NVDA onto read-only media such as a CD, you should just copy the download package.
+Running the portable version directly from read-only media is not supported at this time.
+Using the temporary copy of NVDA is also an option (e.g. for demonstration purposes), though starting NVDA in this way each time can become very time consuming.
+
+Apart from the inability to automatically start during and/or after log-on, the portable and temporary copies of NVDA also have the following restrictions:
+-	The inability to interact with applications running with administrative privileges, unless of course NVDA itself has been run also with these privileges (not recommended).
+-	The inability to read User Account Control (UAC) screens when trying to start an application with administrative privileges.
+-	Windows 8 and later: the inability to support input from a touchscreen.
+-	Windows 8 and later: the inability to provide features such as browse mode and speaking of typed characters in Windows Store apps.

```

Comment:
I'm not sure why these tabs are included - but it is inconsistent with other lists

```suggestion
- The inability to interact with applications running with administrative privileges, unless of course NVDA itself has been run also with these privileges (not recommended).
- The inability to read User Account Control (UAC) screens when trying to start an application with administrative privileges.
- Windows 8 and later: the inability to support input from a touchscreen.
- Windows 8 and later: the inability to provide features such as browse mode and speaking of typed characters in Windows Store apps.
```

Reply:
This is still an issue across the entire document

Reply:
I've replaced all the single ` with double ``.  Sorry, still getting used to this formatting!